superset-frontend/tsconfig.json

```diff
@@ -33,20 +24,42 @@
       ],
       "@superset-ui/plugin-chart-*": ["./plugins/plugin-chart-*/src"],
       "@superset-ui/preset-chart-*": ["./plugins/preset-chart-*/src"],
```

Comment:
I believe L15-26 can be updated to

```ts
      "@superset-ui/*": [
        "./node_modules/@superset-ui/*/src",
        "./node_modules/@superset-ui/*",
      ],
```

Reply:
unfortunately, I got this error when I make this change. I will make a separate PR to improve it. 

![image](https://user-images.githubusercontent.com/2016594/144694371-9fa80ee7-f1e9-41dd-bae0-adf6554e62cd.png)